cmake/find/Findreadline.cmake

```diff
@@ -0,0 +1,20 @@
+
+# The readline library is autotools-based, but does not include
+# pkgconfig files.  So, a find module is needed to locate the outputs.
```

Comment:
It seems that version 8.0 has `*.pc` files.

Reply:
I'd still like to use the 6.3 version.

Reply:
The thing about writing `Find*.cmake` modules is that they are either hard or even impossible to create correctly. E.g., as you said instead of "ncursesw" user can specify something else. Or a user may want to build a shared library, and the name will be `libreadline.so` instead of `libreadline.a`. I've tried a test with `HUNTER_BUILD_SHARED_LIBS=ON`, and it picks a `/usr/lib/x86_64-linux-gnu/libreadline.a` from the system instead of the library from Hunter root because there is only `~/.hunter/.../Install/lib/libreadline.so` present.